src/pymedphys/dcm/_level1/dcmdose.py

```diff
@@ -365,7 +286,8 @@ def extract_iec_fixed_coords(dcm):
        https://arxiv.org/ftp/arxiv/papers/1406/1406.0014.pdf
     """
 
-    check_dcmdose(dcm)
+    if dcm.Modality != "RTDOSE":
+        raise ValueError("The input DICOM file is not an RT Dose file")
```

Comment:
This here seems repeated a few times. I think pulling this out into a function would be of value to keep things DRY.

Reply:
Alright :)